### ISSUE TITLE
Cross Browser Testing 

### ISSUE THREAD

**Issue by MikeWalrath** [hpe-design/design-system#53]
**Browsers / OS**
- [x] Screen reader test
- [x] Edge
  - [x] MacOS
  - [x] Windows 10
- [x] IE11
  - [x] Windows 10
  - [x] Windows Phone
- [x] Firefox
  - [x] Android
  - [x] iOS
  - [x] MacOS
  - [x] Windows 10
- [x] Chrome
  - [x] Android
  - [x] iOS
  - [x] MacOS
  - [x] Windows 10
- [x] Safari
  - [x] iOS
  - [x] MacOS
- [x] Opera
  - [x] iOS
  - [x] MacOS
  - [x] Windows 10

**Comment by halocline** [comment]
Testing Script:
- Go to Home Page
  - Should be able to use keyboard to tab through entirety of page contents
  - Use keyboard to navigate to ThemeMode Toggle
    - Should be able to switch between Light/Dark themes
  - Use keyboard to navigate to SearchInput
    - Should be able to use arrow keys to browse search suggestions
    - Should be able to ESC and have focus return to SearchInput
  - Use keyboard to navigate to "Foundation" tile
    - Should be able to ENTER and be taken to Foundation Page
- On Foundation Page
  - Should be able to use keyboard to tab through entirety of page contents
  - Use keyboard to navigate to "Branding" list item
    - Should be able to ENTER and be taken to Branding Page
- On Branding Page
  - Should be able to use keyboard to tab through entirety of page contents
  - Use keyboard to navigate to SearchInput
    - Type 'button' and ENTER
    - Should be taken to the Controls page
- On Controls Page
  - Use keyboard to navigate to "show code" control
  - ENTER
  - Code sample should expand and display code sample

**Comment by halocline** [comment]
**Results**

TLDR; Site does not load on IE11. Minor visual bugs related to ThemeToggle button on mobile and CodeSample on mobile. Issues being created for each.

Summary: https://docs.google.com/spreadsheets/d/1uhF9cOTVYPjvU4p3HF5LyyTSyX2uodoioodJ7OOMmNg/edit?usp=sharing
